Dart/src/com/jetbrains/lang/dart/ide/devtools/DevToolsInstance.kt

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2020 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
```

Comment:
```
// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
```

Reply:
Sorry, I copied this uncarefully from the flutter plugin